rules/utils/get-docs-url.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+const {basename} = require('path');
+
+const repoUrl = 'https://github.com/sindresorhus/eslint-plugin-unicorn';
+
+module.exports = function (ruleName, commitHash) {
+	ruleName = ruleName || basename(module.parent.filename).replace('.js', '');
```

Comment:
Just pass `.js` as second argument to `path.basename()`. See https://nodejs.org/api/path.html#path_path_basename_path_ext